images/android/v3.8-python-jira-git/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM python:3.8
```

я бы взял слим бустер, если не критично, он меньше весит + быстрее собираться будет
```suggestion
FROM python:3.8-slim-buster
```